Created a Seeder File and name it as UserSeeder

YANCZY opened the pull request:
The following files that has been added are:
**DatabaseSeeder.php** (database/seeders/DatabaseSeeder.php)
**UserSeeder.php** (database/seeders/UserSeeder.php)

In the UserSeeder I added a script to create a user with random details. And UserSeeder is being called in DatabaseSeeder.

MarJose123 reviewed (commented):
missing `UserSeeder`

MarJose123 reviewed (changes requested):
Add the missing `UserSeeder` Class

YANCZY commented:
I have added the UserSeeder into the commit.

Please also see the following files I have added

RoleSeeder - For Roles and Permission
Kernel - Add a middleware package of spatie

Also in the password of user is password 123 and everything else is we're using faker library.

MarJose123 commented on config/app.php:
Not needed, please remove.

MarJose123 reviewed (changes requested):
Please remove the `  Spatie\Permission\PermissionServiceProvider::class,` in the `config/app.php`
